### ISSUE TITLE
Question about t=0 LLaVA 665k fine-tuning step

### ISSUE THREAD

**Issue by TL-QZ** [adymaharana/adapt-inf#2]
Dear authors,

Congratulation on your publication and thank you for sharing the source code of the adapt-infinity method. I want to ask what is the starting model of your first finetuning step on the t=0 llava665k model. Did you use LLaVA-1.5-7B-LoRA directly or did you train t=0 step model upon the pre-trained projection layers + Vecuna-7B (LLaVA stage 1 trained models) and using adapt-infinity to select 25K samples from 665k dataset to replace the original LLaVA stage 2 finetuning. If it is the latter, did you use gradients from the full Vecuna-7B model? Because that is 200M parameters per layer which is requires 48 gb gpu memory ot initialize the TRAK random projector.

I understand you must be busy and it will take sometime to update the selection and training command, but it would be great if you could provide some more contexts.

Thanks again
